### PR TITLE
Show Correct StackTrace

### PR DESCRIPTION
**Pull request by mdashti**
In the current version, the stack-trace printed to the XML output is incorrect. This PR fixes this issue.

### CODE REVIEW

File: src/to_xml.jl [hunk 1]
```diff
@@ -129,7 +129,8 @@ end
 
 function to_xml(v::Error)
     buff = IOBuffer()
-    Base.show_backtrace(buff, scrub_backtrace(backtrace()))
+    print(buff, "\n")
+    join(buff, ("  " * line for line in split(v.backtrace, "\n")), "\n")
     backtrace_str = String(take!(buff))
 
     x_testcase = error_xml(string(v.value), typeof(v.value), backtrace_str)
```
Comment (mmiller-max) [inline src/to_xml.jl:136]: I think we can get the same information in the report by just doing:

```suggestion
    x_testcase = error_xml(string(v.value), typeof(v.value), v.backtrace)
```

And deleting all the `buff` lines

**Review by mmiller-max — CHANGES_REQUESTED** [review]
Good spot, thanks! I think we can simplify things slightly before merging - please see comment.

File: src/to_xml.jl (hunk 1 above)
Comment (mdashti) [reply src/to_xml.jl:136]: That was my first guess, too. But it doesn't work. You can give it a try. That's why I had to use the `buff` lines.

Comment (mmiller-max) [reply src/to_xml.jl:136]: It works for me, testing the `ErroredTest` test package (`test/test_packages/ErroredTest`) gives (after pretty printing):
```
<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="" id="_id_" tests="0" failures="0" errors="1">
    <testsuite name="Top level tests" id="_id_" tests="0" failures="0" errors="1">
        <error message="UndefVarError(:function_doesnt_exist)" type="String">UndefVarError: function_doesnt_exist not defined
Stacktrace:
 [1] top-level scope at /Users/malmiller/repos/TestReports.jl/test/test_packages/ErroredTest/test/runtests.jl:4
 [2] include(::String) at ./client.jl:457
 [3] top-level scope at none:13
 [4] top-level scope at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Test/src/Test.jl:1115
 [5] top-level scope at none:13
</error>
    </testsuite>
</testsuites>
```

Can you show an example where it doesn't work?

Comment (mdashti) [reply src/to_xml.jl]: My bad. Your suggested change also works. Pushed a fix for it.

**Review by mmiller-max — APPROVED** [review]
This is good, thanks for contributing!

**Pull request merged**